server/src/test/java/org/apache/iotdb/db/integration/IoTDBSimpleQueryIT.java

```diff
@@ -32,11 +32,7 @@
 import org.junit.Before;
 import org.junit.Test;
 
-import java.sql.Connection;
-import java.sql.DriverManager;
-import java.sql.ResultSet;
-import java.sql.SQLException;
-import java.sql.Statement;
+import java.sql.*;
 import java.util.ArrayList;
```

Comment:
Hi, using a wild card is not a good choice. Don't change this.

Reply:
okk